### ISSUE TITLE
Page focus should start on the first die

### ISSUE THREAD

**Issue by irilyth** [buttonmen-dev/buttonmen#1020]
When the game page loads, the page focus should be set to be on the first die, for keyboard navigation purposes.

(Or, perhaps, the page element immediately before the first die, so that tab takes you _to_ the first die; I don't care which.)
